sdk/python/tests/test_feature_store.py

```diff
@@ -72,6 +72,24 @@ def feature_store_with_gcs_registry():
     )
 
 
+@pytest.fixture
+def feature_store_with_s3_registry():
```

Comment:
Can you add test cases with this fixture? See how other fixtures are used in this file.

Reply:
done